README.md

```diff
@@ -1,5 +1,8 @@
 # Kresus for YunoHost
 
+[![Integration level](https://dash.yunohost.org/integration/kresus.svg)](https://ci-apps.yunohost.org/jenkins/job/kresus%20%28Community%29/lastBuild/consoleFull)  
```

Comment:
link should be: https://ci-apps.yunohost.org/ci/apps/kresus/

Reply:
I'm asking to the example_ynh maintainer, what is the best to use:
https://github.com/YunoHost/example_ynh/commit/8a2330837be2ad231cc4c6fc316ec65455b7eb23#r32002502